test/markup/php/comments.txt

```diff
@@ -17,3 +17,12 @@ function isEven($a) {
 function isOdd($a) {
     return ($a % 2) === 1;
 }
+
```

Comment:
Please add tests to associated files.  `comments` is meant to test comment highlighting.  It seems we now need a `functions` markup tests added to test function highlighting.  This new test should be placed there - in `functions.txt` and `functions.expect.txt`.

Reply:
Done